src/schema/comments.ts

```diff
@@ -206,6 +212,21 @@ export const typeDefs = /* GraphQL */ `
   }
 
   extend type Query {
+    """
+    Get the comments feed
+    """
+    commentFeed(
+      """
+      Paginate after opaque cursor
+      """
+      after: String
+
+      """
+      Paginate first
+      """
+      first: Int
+    ): CommentConnection!
```

Comment:
Should this be only for auth users?

Reply:
Hmm at this point it is yes, let me add that quickly :)